File: pset6/mario/less/mario.py
from cs50 import get_int

# Prompt for a pyramid between 1 and 8
while True:
    height = get_int("Height: ")

    if height > 0 and height < 9:
        break

# build pyramid
for i in range(0, height, 1):
    print(" " * (height - 1 - i), end="")
    for j in range(0, i + 1, 1):
        print("#", end="")
    print()

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void blocks(int n);
void spaces(int n);

int main(void)
{
    //Prompt user to define pyramid height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    
    //Using height loop until pyramid built
    for (int i = 0; i < height; i++)
    {
        //Loop to align right
        for (int k = height - i; k > 1; k--)
        {
            printf(" ");
        }
        printf("#");
        for (int j = 0; j < i; j++)
        {            
            printf("#");
        }
        printf("  ");
        //Loop for right side
        for (int l = 0; l < i + 1; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}